### PR TITLE
Updated required Ubuntu/Debian packages

### PR DESCRIPTION
**Pull request by notz**
* replaced python mysql package
* added missing django restframework

### CODE REVIEW

File: docs/admin/install.rst [hunk 1]
```diff
@@ -63,11 +63,11 @@ install them you can use apt-get:
         python-whoosh python-pil python-libravatar \
         python-babel Git mercurial python-social-auth \
         python-django-compressor python-django-crispy-forms \
-        python-dateutil
+        python-djangorestframework python-dateutil
 
     # Optional for database backend
 
-    apt-get install python-mysqldb   # For MySQL
+    apt-get install python-pymysql   # For MySQL
```
Comment (nijel) [inline docs/admin/install.rst]: Using package which is not available in stable is not good idea.

Comment (notz) [inline docs/admin/install.rst]: Why? It's in 16.04 - and not working with the other package

Comment (nijel) [inline docs/admin/install.rst]: Why it's not working with other package? I don't see any problems with it on stable, testing and unstable...

Comment (notz) [inline docs/admin/install.rst]: I got this on Ubuntu 16.04 with python-mysqldb installed:

`Traceback (most recent call last):
  File "/var/www/weblate/weblate/wsgi.py", line 44, in <module>
    application = get_wsgi_application()
  File "/usr/lib/python2.7/dist-packages/django/core/wsgi.py", line 14, in get_wsgi_application
    django.setup()
  File "/usr/lib/python2.7/dist-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/lib/python2.7/dist-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/usr/lib/python2.7/dist-packages/django/apps/config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/lib/python2.7/dist-packages/django/contrib/auth/models.py", line 41, in <module>
    class Permission(models.Model):
  File "/usr/lib/python2.7/dist-packages/django/db/models/base.py", line 139, in __new__
    new_class.add_to_class('_meta', Options(meta, **kwargs))
  File "/usr/lib/python2.7/dist-packages/django/db/models/base.py", line 324, in add_to_class
    value.contribute_to_class(cls, name)
  File "/usr/lib/python2.7/dist-packages/django/db/models/options.py", line 250, in contribute_to_class
    self.db_table = truncate_name(self.db_table, connection.ops.max_name_length())
  File "/usr/lib/python2.7/dist-packages/django/db/__init__.py", line 36, in __getattr__
    return getattr(connections[DEFAULT_DB_ALIAS], item)
  File "/usr/lib/python2.7/dist-packages/django/db/utils.py", line 241, in __getitem__
    backend = load_backend(db['ENGINE'])
  File "/usr/lib/python2.7/dist-packages/django/db/utils.py", line 112, in load_backend
    return import_module('%s.base' % backend_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/lib/python2.7/dist-packages/django/db/backends/mysql/base.py", line 23, in <module>
    import pymysql
`

After installing python-pymysql it worked

Comment (nijel) [inline docs/admin/install.rst]: Okay, this is Ubuntu specific patch, so we most likely need different instructions for Debian than for Ubuntu, as Django package in Debian doesn't work with python-pymysql...

Quoting Ubuntu package changelog:

>    - debian/patches/pymysql-replacement.patch: Use pymysql as drop in
>      replacement for MySQLdb.
>    - debian/control: Drop python-mysqldb in favor of python-pymysql.

Comment (okwow123) [inline docs/admin/install.rst]: hello, guys~
could you help me? with my heroku problem...

**Comment by nijel** [comment]
Thenks, merged.

File: docs/admin/install.rst (hunk 1 above)
Comment (nijel) [inline docs/admin/install.rst]: @okwow123 Try opening separate issue for that...

**Pull request merged**